code/fireball/fireballs.h

```diff
@@ -95,8 +95,7 @@ typedef struct fireball {
 } fireball;
 // end move
 
-extern fireball Fireballs[MAX_FIREBALLS];
-extern int Num_fireballs;
+extern SCP_vector<fireball> Fireball_vec;
```

Comment:
The name should stay as `Fireballs` for consistency with the rest of the code. That `_vec` suffix isn't really necessary.